onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -27,6 +28,64 @@ void HandleActivationNodeEdges(Graph& g, const Node& act, Node& fused_conv) {
   }
 }
 
+// get min/max values from Clip if they are constant. Returns false if mutable and cannot be used
+static bool GetClipConstantMinMax(const Graph& graph, const Node& node, float& min, float& max) {
+  min = std::numeric_limits<float>::lowest();
+  max = std::numeric_limits<float>::max();
+
+  // Clip opset 6 has min and max as attributes. they're inputs from opset 11 on.
+  bool min_max_are_attributes = graph_utils::IsSupportedOptypeVersionAndDomain(node, "Clip", {6});
+  bool can_use = true;
+
+  if (min_max_are_attributes) {
+    min = graph_utils::GetNodeAttribute(node, "min")->f();
+    max = graph_utils::GetNodeAttribute(node, "max")->f();
+  } else {
+    // update min/max if provided via a constant initializer
+    // return true if value is default or coming from a constant initializer
+    // return false if value is mutable
+    auto get_constant_value = [&graph](const Node& node, size_t input_idx, float& value) {
```

Comment:
maybe rename to is_constant_value or something like that since it just returns boolean?

Reply:
and/or mention that value is an output parameter (although the lack of const says it already)